bin/public/css/style.css

```diff
@@ -1,28 +0,0 @@
-div {
```

Comment:
I kept those files on the other stylesheet, as well. I though keeping same styles in both will only make waiting/loading time longer, but i understand your point.